NSS-Day-Cohort-66/rare-api-golden-droids
# RE: ticket #1 - Register new User

I added a register method and serializer to UserViewSet

Supported route:
`/register` for a POST request that returns a single object of the newly created token

***TO TEST***
1. Fetch the branches
2. Checkout `srt-work` branch
3. Start debug session
4. Send a POST to `http://localhost:8000/register` with the following JSON body or something similar:
```
{
    "username": "megcharmed",
    "email": "meg@ducharme.com",
    "password": "ducharme",
    "first_name": "Meg",
    "last_name": "Ducharme"
}
```
5. Should receive a 201 status and a similar looking response body:
```
{
    "token": "c8fb7595d4416fe2c7c62d8f550568153cdb798c"
}
```

OR you can boot up the client, nav to /register, fill out the form provided to us, and use that submit button to make your post of whatever info you fill in. Should still receive 201 and new token as response.


Closes ticket #1
